Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserViewUtils.cpp

```diff
@@ -276,6 +276,27 @@ namespace AzToolsFramework
                                 canDelete = false;
                             }
                         }
+                        else
+                        {
+                            QMessageBox warningBox;
+                            warningBox.setWindowTitle(QObject::tr(isFolder ? "Folder Deletion - Warning" : "Asset Deletion - Warning"));
+                            warningBox.setText(
+                                QObject::tr("O3DE is unable to detect if this file is being used inside a level, prefab, or asset."
+                                            "By deleting these asset(s), you understand this might break a connection if it's still in use."));
+                            warningBox.setInformativeText(
+                                QObject::tr("Currently, delete is a permanent action and cannot be undone.\n"
+                                    "Do you wish to proceed with this deletion?"));
+                            warningBox.setIcon(QMessageBox::Warning);
+                            warningBox.addButton(QMessageBox::Cancel);
+                            warningBox.addButton(QObject::tr("Delete"), QMessageBox::YesRole);
+                            warningBox.setDefaultButton(QMessageBox::Yes);
+                            warningBox.setFixedWidth(600);
```

Comment:
Just curious was there a reason you had to restrict the width? Was it not getting auto-sized properly?

Reply:
I restricted the width to 600 since all the other message boxes in this file are fixed width message boxes at 600 as well, I'm not exactly sure why that's the case though